assets/src/scripts/components/Layout/Sidebar.tsx

```diff
@@ -21,8 +21,8 @@ export default function Sidebar({
 }) {
   return (
     <Col className="builder__sidebar" md="3">
-      {isEmptyCodelist ? (
-        <>{isEditable && <SearchForm />}</>
+      {isEmptyCodelist && isEditable ? (
+        <SearchForm />
       ) : (
```

Comment:
Does this do the same thing? I'm struggling a bit, but if `isEmptyCodelist` is `true` and `isEditable` is `false`, then previously the block would return an empty block `<></>`, but in the change it now returns the bit after the `:` of the ternary operator. Have I read that correctly?

Reply:
Fixed this in the latest commit